test/t8n/t8n.cpp

```diff
@@ -5,6 +5,7 @@
 #include "../state/errors.hpp"
 #include "../state/ethash_difficulty.hpp"
 #include "../state/mpt_hash.hpp"
+#include "../state/requests.hpp"
```

Comment:
can drop sha256.hpp :point_down: and (I think so) the changes in  test/t8n/CMakeLists.txt that allowed the include